frontend/src/components/elements/external-link-arrow.tsx

```diff
@@ -0,0 +1,5 @@
+export default function ExternalLinkIcon() {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" className="pl-2" height="70%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-move-up-right"><path d="M13 5H19V11"/><path d="M19 5L5 19"/></svg>
```

Comment:
```suggestion
        <svg xmlns="http://www.w3.org/2000/svg" className="pl-2 lucide lucide-move-up-right" height="70%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M13 5H19V11"/><path d="M19 5L5 19"/></svg>
```
Il faut utiliser className et non class, de plus tu avais déjà un champs className, j'ai juste déplacé tes classes CSS, vérifie si c'est bien le rendu que tu veux donner à ton élément